frontend/src/index.ts

```diff
@@ -283,90 +283,104 @@ export class App extends LiteElement {
     const isSuperAdmin = this.appState.userInfo?.isSuperAdmin;
     let homeHref = "/";
     if (!isSuperAdmin && this.appState.orgSlug) {
-      homeHref = `/orgs/${this.appState.orgSlug}`;
+      homeHref = this.orgBasePath;
     }
 
+    const showFullLogo =
+      this.viewState.route === "login" ||
+      !this.authService.authState ||
+      new URLSearchParams(window.location.search).get("mode") === "demo";
+
+    console.log("showFullLogo:", showFullLogo);
+
     return html`
       <div class="border-b bg-neutral-50">
         <nav
           class="mx-auto box-border flex h-12 items-center justify-between px-3 xl:pl-6"
         >
-          <a
-            class="items-between flex gap-2"
-            aria-label="home"
-            href=${homeHref}
-            @click=${(e: MouseEvent) => {
-              if (isSuperAdmin) {
-                this.clearSelectedOrg();
-              }
-              this.navLink(e);
-            }}
-          >
-            <img class="h-6" alt="Browsertrix logo" src=${brandLockupColor} />
+          <div class="flex items-center">
+            <a
+              class="items-between flex gap-2"
+              aria-label="home"
+              href=${homeHref}
+              @click=${(e: MouseEvent) => {
+                if (isSuperAdmin) {
+                  this.clearSelectedOrg();
+                }
+                this.navLink(e);
+              }}
+            >
+              <div
+                class="${showFullLogo
+                  ? "w-48"
+                  : "w-7"} h-7 bg-cover bg-no-repeat"
+                style="background-image: url(${brandLockupColor})"
+                role="img"
+                title="Browsertrix logo"
+              ></div>
+            </a>
+            ${when(
+              this.authService.authState,
+              () => html`
+                <div role="separator" class="ml-2.5 h-6 w-0 border-l"></div>
+                ${this.renderOrgs()}
+              `,
+            )}
+          </div>
+          <div class="grid auto-cols-max grid-flow-col items-center gap-5">
             ${isSuperAdmin
-              ? html`<btrix-tag>${msg("Admin")}</btrix-tag>`
```

Comment:
I think this might be worth keeping? Was added recently so that we know at a glance when we're logged in as a superadmin.

Reply:
Honestly I might even push for this being a banner at the top of the screen that's always visible

Reply:
Could go either way! Maybe the simplest thing to do now is just to add the badge back next to the org name, and we can reassess separately? (just because I know we're trying to get self-sign up launched quickly)

Reply:
Yeah totally! Out of scope for this PR, i think

Reply:
I had renamed the superadmin "Dashboard" link to "Admin Dashboard" so the badge seemed redundant, do you think it's a placement thing where it's helpful to see at a glance in the top-left?

Reply:
How about something like this? Link to admin dashboard
<img width="1512" alt="Screenshot 2024-08-06 at 4 23 33 PM" src="https://github.com/user-attachments/assets/8e208df9-6e1a-4a7c-97fe-181a5aba8fe1">

Reply:
That looks nice! I think the goal with the badge was primarily to make sure we know at a quick glance when we're logged in as superadmin, which the "Admin Dashboard" text might have been a bit too subtle for.